src/main/java/org/terasology/climateConditions/FrostbiteSystem.java

```diff
@@ -65,7 +65,7 @@ public void hypothermiaLevelChanged(HypothermiaLevelChangedEvent event, EntityRe
         //Frostbite Effect remains active for Hypothermia Levels 3 and greater.
         if(newLevel == 3 && oldLevel < newLevel) {
```

Comment:
We don't have strict code style guidelines, but I believe it's more common to have a space between `if` and the opening brance:
```suggestion
        if (newLevel == 3 && oldLevel < newLevel) {
```